### ISSUE TITLE
FAQ Page

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#784]
# Description
Replaces #559.

New mockups in Figma:
https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1%3A2

Draft FAQ content here:
https://docs.google.com/document/d/15g8psA9-w6BCEj-JxSNuLgSf0BEqLvjXrlMFK2i8EGk/edit

## Action Items

**Priority:**
- [x] Implement list of FAQs that can be collapsed/expanded on mouseclick or via keyboard.
- [x] Text-only FAQs should be a single reusable component (rather than repeated hardcoded jsx). 

- [x] Add `/faq` route to Routes.jsx that will render the overall FAQ (or whatever name it is given) component.
- [x] Add a `<NavLink>` in the header that navigates to `/faq`

**Would be super awesome to have:**
- [x] Search bar. This could search all text in all questions or define a list of keywords for each question if all text search returns overly broad results.
- [x] Including images in relevant FAQ answers.

## Resources

https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1%3A2
https://docs.google.com/document/d/15g8psA9-w6BCEj-JxSNuLgSf0BEqLvjXrlMFK2i8EGk/edit

![Screen Shot 2020-08-19 at 4 09 26 PM](https://user-images.githubusercontent.com/40484278/90698892-c20d8a00-e236-11ea-9506-93fb5fcef8bc.png)
![Screen Shot 2020-08-19 at 4 09 47 PM](https://user-images.githubusercontent.com/40484278/90698896-c3d74d80-e236-11ea-87ab-1867839a858a.png)

**Comment by hannahlivnat** [comment]
The current mockups in figma are with all-text content, would it be possible to get mockups incorporating the image / text layout for the faq that have one or multiple images? @adamkendis

**Comment by adamkendis** [comment]
@danarchen 
Any chance your team has bandwidth available to mock up a few of the FAQs that include images? 

@hannahlivnat 
Let's see what Dana says. If they're busy this week, feel free to move forward using your best judgment.

**Comment by danarchen** [comment]
I'll ask Yuenyi to see if she can get something by this Thursday. 
She didn't get access to the FAQ document till last week...

**Comment by hannahlivnat** [comment]
@danarchen @adamkendis -- The mockups with the expanded faqs look great! I had one question for the top menu bar images. The text within the mockup is different from the text that's on our live site. 

Example: 'Explore My 311 Data' vs. 'Explore My Council's 311 Data', 'Compare Councils' vs. 'Compare Different Councils'

Should I stick with the text from the Figma or take a screenshot from the current site?

**Comment by adamkendis** [comment]
@hannahlivnat - Could you take new menu bar screenshots from the current site? We should keep the FAQs consistent with what we have live. We'll just need to remember to update the FAQs if we make any changes.

**Comment by hannahlivnat** [comment]
@adamkendis will do, thank you!

**Comment by hannahlivnat** [comment]
**Issue update:** 
- Currently have basic q&a layout from figma and able to filter through questions by search. 
![Screen Shot 2020-09-09 at 4 22 45 PM](https://user-images.githubusercontent.com/54752231/92664622-5197dd00-f2b9-11ea-915d-96161e0d3ba0.png)
![Screen Shot 2020-09-09 at 4 23 02 PM](https://user-images.githubusercontent.com/54752231/92664626-5361a080-f2b9-11ea-95a9-adb357351682.png)


**Working On:** 
- Autocomplete feature for search bar
- Make toggle keyboard accessible
- figuring out an error that keeps logging from search -- Unchecked runtime.lastError: The message port closed before a response was received.

**Timeline:**
- Final interview is next Thursday. Will be back at meetings following interview, should have this page done in next 2 weeks.
